modules/imaging_uploader/README.md

```diff
@@ -12,3 +24,77 @@ max_execution_time = 10800
 upload_max_filesize = 1024M
 post_max_size = 1024M
 ```
+
+### Uploaded file
+
+The uploaded file is expected to be of one of the following types: 
+`.tgz, .tar.gz or .zip`. The filename should follow the:
+`PSCID_CandID_VisitLabel_*OptionalSuffix*.zip` naming convention for a `.zip` 
+file. It is also expected that the candidate and visit are already created in 
+the database.
```

Comment:
Wouldn't it be more appropriate to put this this be in the front end and/or help text rather than here? It's the logged in users who are trying to use the module that need this information.

On the other hand, it does also make sense here and I don't see any section in the imaging browser template that would be more appropriate. Maybe just rename the section to something like 'Uploaded File Requirements'